### DISCUSSION TITLE
bu tokeni elde tutma

### DISCUSSION THREAD

**Discussion by ruzgarkafa** [IBAX-io/go-ibax#1241] [Ideas]
bu tokeni birkaç ay elimizde tutarsak tahmini kac x yapar tahminleri olan varmıdır

**Comment by erhanqwerty** [comment]
sadece tahmin: önce 40x yukarı, 30x aşağı. yatay gider. 6 ay sonra uçabilir.

**Comment by sedat2855** [comment]
BENCE  İLK PUMPTA SATILMALI
